pkg/controllers/dlb/controller.go

```diff
@@ -33,7 +33,9 @@ import (
 	"github.com/pkg/errors"
 )
 
-const ownerKey = ".metadata.controller.dlb"
+const (
+	ownerKey = ".metadata.controller.dlb"
+)
```

Comment:
this change doesn't make sense

Reply:
This change was made since there were two consts there and I removed amd64 part.
But, I kept as it is since.. all of the others also have the same structures.
Or would I change all of the others to be
> const ownerKey = ".metadata.controller.-"
form?